iphone/TitaniumKit/TitaniumKit/Sources/API/TiUIView.m

```diff
@@ -385,9 +385,36 @@ - (void)frameSizeChanged:(CGRect)frame bounds:(CGRect)bounds
   if (backgroundRepeat) {
     [self renderRepeatedBackground:backgroundImage];
   }
+
+  if ([self hasCustomBorderRadii]) {
+    __block UIRectCorner corners;
+    NSArray<NSNumber *> *edges = [proxy valueForKey:@"borderRadiusCorners"];
+    TiDimension borderRadius = TiDimensionFromObject([proxy valueForKey:@"borderRadius"]);
+
+    if (TiDimensionIsUndefined(borderRadius)) {
+      [self throwException:@"Cannot create custom border radius" subreason:@"The border radius is invalid" location:CODELOCATION];
```

Comment:
@garymathews  In iOS , exception is thrown if property `borderRadiusCorners ` is set without setting `borderRadius `. How is behavior in Android?
@sgtcoolguy Should we throw exception in this case?
Or Log a warning message and ignore it? 
Or Ignore it silently?
I am in favor of logging a warning message.